packages/eslint-plugin-lit-a11y/tests/lib/rules/aria-attr-valid-value.js

```diff
@@ -109,6 +109,9 @@ ruleTester.run('aria-attr-valid-value', rule, {
     { code: 'html`<div aria-labelledby="additions additions" />`' },
     { code: 'html`<div aria-labelledby="additions removals text" />`' },
     { code: 'html`<div aria-labelledby="additions removals text all" />`' },
+
+    // FUNCTION CALL
+    { code: 'html`<div aria-label="${foo("foo")}"></div>`' },
```

Comment:
Could you add a test for:
```js
    { code: 'html`<div aria-label="${this.foo("foo")}"></div>`' },
```

as well? Just to make sure 🙂

Reply:
Sure. 
Just added and test passed as well :D